scalableminds/webknossos
# fix initial color of user bounding boxes

This PR fixes that the initial displayed color of a user bounding box in the settings is not correct.

### URL of deployed dev instance (used for testing):
- https://fixinitialuserbbcolor.webknossos.xyz

### Steps to test:
- Open a dataset in view mode and add a user bounding box. The color of the box in the settings should match the color displayed in the viewports.

### Issues:
- fixes #4699

------
